src/core/transport/chttp2/frame_window_update.c

```diff
@@ -36,9 +36,10 @@
 
 #include <grpc/support/log.h>
 
-gpr_slice grpc_chttp2_window_update_create(uint32_t id,
-                                           uint32_t window_update) {
+gpr_slice grpc_chttp2_window_update_create(
+    uint32_t id, uint32_t window_update, grpc_transport_one_way_stats *stats) {
   gpr_slice slice = gpr_slice_malloc(13);
+  stats->header_bytes += 13;
```

Comment:
Same here I think const value is easier to avoid mistakes when we change the value.